dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/ProcessDefinitionServiceImpl.java

```diff
@@ -1876,6 +1878,27 @@ protected void doBatchOperateProcessDefinition(User loginUser,
         }
     }
 
+    /**
+     * get new Task name or Process name when copy or import operate
+     * @param originalName Task or Process original name
+     * @param suffix "_copy_" or "_import_"
+     * @return
+     */
+    private String getNewName(String originalName, String suffix) {
```

Comment:
can we add some test for test function? maybe we can test the public function use it.
BTW, how about the process definition contain `_copy_` or `_import_` in the name? like `wf_copy_all_ods_data_to_dw`

Reply:
use regex match